elastic_search/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.15.0"
+  required_providers {
```

Comment:
I thought we were removing provider info from terraform modules? Is this required?

Reply:
These are required versions not providers